conformance/BUILD.bazel

```diff
@@ -24,47 +31,60 @@ ALL_TESTS = [
     "@com_google_cel_spec//tests/simple:testdata/wrappers.textproto",
 ]
 
-sh_test(
-    name = "ct",
-    srcs = ["@com_google_cel_spec//tests:conftest.sh"],
-    args = [
-        "$(location @com_google_cel_spec//tests/simple:simple_test)",
-        "--server=$(location //server/main:cel_server)",
-        # Tests that need to be removed as the spec has changed
-        "--skip_test=comparisons/eq_literal/eq_mixed_types_error,eq_list_elem_mixed_types_error,eq_map_value_mixed_types_error;ne_literal/ne_mixed_types_error",
-        "--skip_test=comparisons/in_list_literal/elem_in_mixed_type_list_error",
-        "--skip_test=comparisons/in_map_literal/key_in_mixed_key_type_map_error",
-        "--skip_test=macros/exists/list_elem_type_exhaustive,map_key_type_exhaustive",
+_TESTS_TO_SKIP = [
+    "comparisons/eq_literal/eq_mixed_types_error,eq_list_elem_mixed_types_error,eq_map_value_mixed_types_error",
+    "comparisons/ne_literal/ne_mixed_types_error",
+    "comparisons/in_list_literal/elem_in_mixed_type_list_error",
+    "comparisons/in_map_literal/key_in_mixed_key_type_map_error",
+    "macros/exists/list_elem_type_exhaustive,map_key_type_exhaustive",
 
-        # Failing conformance tests.
-        "--skip_test=fields/qualified_identifier_resolution/map_key_float,map_key_null,map_value_repeat_key",
-        "--skip_test=fields/qualified_identifier_resolution/map_value_repeat_key_heterogeneous",
-        "--skip_test=macros/map/map_extract_keys",
-        "--skip_test=timestamps/duration_converters/get_milliseconds",
+    # Failing conformance tests.
+    "fields/qualified_identifier_resolution/map_key_float,map_key_null,map_value_repeat_key",
+    "fields/qualified_identifier_resolution/map_value_repeat_key_heterogeneous",
+    "macros/map/map_extract_keys",
+    "timestamps/duration_converters/get_milliseconds",
 
-        # Future enhancments.
-        "--skip_test=enums/strong_proto2",
-        "--skip_test=enums/strong_proto3",
-    ] + ["$(location " + test + ")" for test in ALL_TESTS],
-    data = [
-        "//server/main:cel_server",
-        "@com_google_cel_spec//tests/simple:simple_test",
-    ] + ALL_TESTS,
+    # Future enhancments.
+    "enums/strong_proto2",
+    "enums/strong_proto3",
+]
+
+go_test(
+    name = "go_default_test",
+    size = "small",
+    srcs = [
+        "conformance_test.go",
+    ],
+    tags = [
+        "manual",
+        "notap",
```

Comment:
Should we put `notap` here?

Reply:
Yes, this test target is not meant to be run directly. It is invoked via the conformance_test macros.